modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -105,13 +122,61 @@ To restore a cluster on Runtime Fabric, create a cluster *without* running the `
 ----
 kubectl scale --replicas=0 -n rtf deployment.apps/agent
 ----
-
++
+[start =5]
++
 . Restore the cluster from the backup: 
 [source,copy]
 ----
 ./rtfctl restore <path_to_backup_file>
 ----
-+
 This process may require several minutes to complete.
 
-For Runtime Fabric, confirm the `rtfctl` binary is present in the current directory and the Kubernetes (kubectl) context is set to the cluster you are restoring to.
\ No newline at end of file
+For Runtime Fabric, confirm the `rtfctl` binary is present in the current directory and the Kubernetes (kubectl) context is set to the cluster you are restoring to.
+
+== Create a Backup and Restore for OpenShift Clusters
+
+In the following example, you have a Runtime Fabric with a set of Mule applications and configurations deployed on cluster (named `cluster-1`) inside an OpenShift cluster. To create a backup and perform a restore, follow these steps:
+
+. Run the backup command: 
++
+[source,copy]
+----
+./rtfctl backup <path_to_backup_file> -n <rtf_namespace>
+----
++
+[NOTE]
+MuleSoft performs the back up based on the detected cluster you have. Though, you can also manually specify the type of back up to perform using the flag `--backup-type` string. The flag options are `auto`, `openshift`, and `full`. With the default `auto` flag option, MuleSoft automatically detects the infrastructure cluster provider to perform the back up.
+[start=2]
+. Shutdown `cluster-1`.
+. Delete all Mule application deployments by running the following command:
++
+[source,copy]
+----
+kubectl delete ns <app_namespace>
+----
++
+Do not remove applications from the Runtime Manager UI.
+[start=4]
+. Remove the Runtime Fabric installation.
+. Remove the Runtime Fabric operator from the OpenShift cluster (optional).
+. Send a `curl` command to the management plane API endpoint to regenerate the activation ID. +
+The Runtime Fabric will show as ready for activation. You can install and activate Runtime Fabric again on a new OpenShift cluster. 
++
+[source,copy]
+----
+curl --location --request PATCH 'https://anypoint.mulesoft.com/runtimefabric/api/organizations/{org-id}/fabrics/{fabric-id}/regenerateActivationData' \
+--header 'Authorization: Bearer {bearer-token}
+----
++
+[start=7]
+. Install the Runtime Fabric operator on `cluster-2` using the regenerated activation data ID. +
+The installation must be on the same Runtime Fabric namespace because all labels on the backup point to that installation.
```

Comment:
```suggestion
The installation must be on the same Runtime Fabric namespace because all labels on the backup point to that installation.
```

If you use `because`, you're missing a part of the sentence.